commit 7540f70df98f5c46feb5fe2257f93f543e5821e5
Author: Ard Biesheuvel <ardb@kernel.org>
Date:   Mon Nov 27 13:00:53 2023 +0100

    arm64: Kconfig: drop KAISER reference from KPTI option description
    
    KAISER is a reference to the KASLR hardening technique that already
    existed before Meltdown happened, and by now, it is sufficiently obscure
    that mentioning it does not actually clarify anything. So remove this
    reference, and replace it with KPTI.
    
    Signed-off-by: Ard Biesheuvel <ardb@kernel.org>
    Acked-by: Mark Rutland <mark.rutland@arm.com>
    Link: https://lore.kernel.org/r/20231127120049.2258650-8-ardb@google.com
    Signed-off-by: Will Deacon <will@kernel.org>

diff --git a/arch/arm64/Kconfig b/arch/arm64/Kconfig
index 7b071a00425d..b67e6934316f 100644
--- a/arch/arm64/Kconfig
+++ b/arch/arm64/Kconfig
@@ -1549,7 +1549,7 @@ config ARCH_FORCE_MAX_ORDER
 	  Don't change if unsure.
 
 config UNMAP_KERNEL_AT_EL0
-	bool "Unmap kernel when running in userspace (aka \"KAISER\")" if EXPERT
+	bool "Unmap kernel when running in userspace (KPTI)" if EXPERT
 	default y
 	help
 	  Speculation attacks against some high-performance processors can